src-test/test-code-2.py

```
@@ -0,0 +1,41 @@
+
+
+
+def _is_in_box(
+        self,
+        aircraft: V2ResponseAcItem,
```

Ruff (0.15.12)

[error] Undefined name `V2ResponseAcItem`

(F821)